tools/src/commands/GenerateBareDiffs.ts

```diff
@@ -0,0 +1,149 @@
+import { Command } from '@expo/commander';
+import spawnAsync from '@expo/spawn-async';
+import chalk from 'chalk';
+import { PromisyClass, TaskQueue } from 'cwait';
+import fs from 'fs-extra';
+import os from 'os';
+import path from 'path';
+
+import { EXPO_DIR } from '../Constants';
+import logger from '../Logger';
+
+type ActionOptions = {
+  check?: boolean;
+};
+
+function allVersions(min, max) {
+  const versions: string[] = [];
+  for (let i = min; i <= max; i++) {
+    versions.push(i.toString());
+  }
+  return versions;
+}
+
+function getReleasedSdkVersionRange() {
+  const packageJson = fs.readJSONSync(path.join(EXPO_DIR, 'packages', 'expo', 'package.json'));
+  const expoPackageVersion = packageJson.version;
+  let maxSdk = parseInt(expoPackageVersion.split('.')[0], 10);
+  // if current version is an alpha or beta, then one version behind is the latest stable release
+  if (expoPackageVersion.includes('alpha') || expoPackageVersion.includes('beta')) {
+    maxSdk--;
+  }
+  return {
+    maxSdk,
+    minSdk: maxSdk - 6,
+  };
+}
+
+function sdkToBranch(sdkVersion: string) {
+  if (sdkVersion === 'unversioned') {
+    return 'main';
+  }
+  return `sdk-${sdkVersion}`;
+}
+
+async function executeDiffCommand(diffDirPath: string, sdkFrom: string, sdkTo: string) {
+  const diffPath = path.join(diffDirPath, `${sdkFrom}..${sdkTo}.diff`);
+
+  const diffCommand = `origin/${sdkToBranch(sdkFrom)}..origin/${sdkToBranch(sdkTo)}`;
+
+  const diff = await spawnAsync(
+    'git',
+    ['diff', diffCommand, '--', 'templates/expo-template-bare-minimum'],
+    {
+      cwd: EXPO_DIR,
+    }
+  );
+
+  fs.writeFileSync(diffPath, diff.stdout);
+}
+
+async function action({ check = false }: ActionOptions) {
+  const taskQueue = new TaskQueue(Promise as PromisyClass, os.cpus().length);
+
+  const diffDirPath = path.join(
+    EXPO_DIR,
+    'docs',
+    'public',
+    'static',
+    'diffs',
+    'template-bare-minimum'
+  );
+
+  try {
+    //const sdkVersions = await readSdkVersions(diffDirPath, sdk);
+    // generate from all other SDK version to the specified SDK version
+    let diffJobs: PromiseLike<any>[] = [];
+
+    const { minSdk, maxSdk } = getReleasedSdkVersionRange();
+
+    const sdkVersionsToDiff = allVersions(minSdk, maxSdk);
+    sdkVersionsToDiff.push('unversioned');
+
+    // clear all versions before regenerating
+    await spawnAsync('rm', ['-rf', diffDirPath]);
```

Comment:
```suggestion
    await fs.remove(diffDirPath);
```

nit: maybe we could use the [fs.remove()](https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove.md)